go.mod

```diff
@@ -81,6 +83,7 @@ require (
 	github.com/docker/docker-credential-helpers v0.8.0 // indirect
 	github.com/docker/go v1.5.1-1.0.20160303222718-d30aec9fd63c // indirect
 	github.com/docker/go-metrics v0.0.1 // indirect
+	github.com/eiannone/keyboard v0.0.0-20220611211555-0d226195f203
```

Comment:
Probably does what we expect, by I wonder we want to rely on a dependency which hasn't been updated for 2 years